task/anchore-cli/0.1/secret.yaml

```diff
@@ -0,0 +1,8 @@
+kind: Secret
+apiVersion: v1
+metadata:
+   name: anchore-cli-secret
```

Comment:
is this a sample secret? If so, can you please move this to other directory like `samples` ?

Reply:
done